README.md

```diff
@@ -48,6 +48,15 @@ In this example, the Jaeger UI is available at http://192.168.122.34.
 
 The official documentation for the Jaeger Operator, including all its customization options, are available under the main [Jaeger Documentation](https://www.jaegertracing.io/docs/latest/operator/).
 
+
+## Command line invocation
+
+To dry-run, experiment or run adapt the generated output with kustomize, command line invocation using the `generate`subcommand is possible. In this example we apply the manifest generated by examples/simple-prod.yaml to the namespace `jaeger-test`:
```

Comment:
It might not be just dry-run some people still prefer to use plain manifest files. This paragraph should tell that `jaeger-operator generate` generates kubernetes manifest file from a given CR.

Reply:
Updated, but I'm not sure if you wanted the introduction sentence at all. Maybe you want to spell our the whole paragraph instead? :)

If anything, maybe we should have an example with kustomize or hint that people should use kustomize instead of manually patching the generated file. This would allow them to run generate with a newer version of the operator to get upgrades later, and hopefully not update/redo the local changes.